sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_models.py

```diff
@@ -221,26 +226,32 @@ def value(self, new_value: str) -> None:
             temp = json.loads(new_value)
             temp["id"] = self.feature_id
             self._value = json.dumps(temp)
-            self.enabled = temp.get("enabled", None)
+            self.enabled = temp.get("enabled", False)
+            self.display_name = temp.get("display_name", None)
+            self.description = temp.get("description", None)
             self.filters = None
             conditions = temp.get("conditions", None)
             if conditions:
                 self.filters = conditions.get("client_filters", None)
         except (json.JSONDecodeError, ValueError):
             self._value = new_value
-            self.enabled = None
+            self.enabled = False
             self.filters = None
 
     @classmethod
-    def _from_generated(cls, key_value: KeyValue) -> Union["FeatureFlagConfigurationSetting", ConfigurationSetting]:
+    def _from_generated(cls, key_value: KeyValue) -> "FeatureFlagConfigurationSetting":
         if key_value is None:
             return key_value
         enabled = None
         filters = None
+        display_name = None
+        description = None
         try:
             temp = json.loads(key_value.value)  # type: ignore
             if isinstance(temp, dict):
                 enabled = temp.get("enabled")
```

Comment:
Do we need temp.get("enabled", False)?

Reply:
I don't think we need, because it has been set a non-None value when creating/setting a configuration setting. Instead, the initial value in line245 need to be `False`.

Reply:
`enabled` is required in [schema](https://github.com/Azure/AppConfiguration/blob/main/docs/FeatureManagement/FeatureFlag.v1.1.0.schema.json#L9), it should not be missing in response.

Reply:
You mean if `enabled` is missing, server treats it as non-FeatureFlag config setting?

Reply:
`enabled` only exists in FeatureFlagConfigurationSetting.

Reply:
Ultimately, it is a ConfigSetting. The schema is just for the value of the setting.

Reply:
Added a default value `False`.